batch.go

```diff
@@ -192,10 +164,25 @@ func (c *Client) weakFetchBatch(ctx context.Context, keys []string, expire time.
 			go func(i int) {
 				defer wg.Done()
 				r, err := c.luaGet(ctx, keys[i], owner)
+				ticker := time.NewTimer(c.Options.LockSleep)
```

Comment:
Why not put ticker inside the for loop? Code can be cleaner

Reply:
reuse & reduce allocation

Reply:
The performance gained in this way should be very very little and can be ignored. Why not using the cleaner way?

Reply:
simplified